integration-tests/kubernetes-service-binding-jdbc/src/test/java/io/quarkus/it/k8ssb/jdbc/FruitsEndpointTest.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.it.k8ssb.jdbc;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class FruitsEndpointTest {
```

Comment:
I purposely don't have a native test for this as the service binding code does not do anything that would prevent it from working in native